modules/cms/lang/ar/lang.php

```diff
@@ -0,0 +1,293 @@
+<?php
+
+return [
+    'cms_object' => [
+        'ملف تالف' => 'Invalid file name: :name. File names can contain only alphanumeric symbols, underscores, dashes and dots. Some examples of correct file names: page.htm, page, subdirectory/page',
```

Comment:
The key needs to remain in English, the value needs to be translated. If you don't have a translation for a given key, then don't include it in the PR.